client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx

```diff
@@ -0,0 +1,48 @@
+// HelloWorldWidget is an arbitrary name for any "dumb" component. We do not recommend suffixing all your
+// dump component names with Widget.
+
+import React, { PropTypes } from 'react';
+import Immutable from 'immutable';
+import _ from 'lodash';
+
+// Simple example of a React "dumb" component
+export default class HelloWorldWidget extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    // Uses lodash to bind all methods to the context of the object instance, otherwise
+    // the methods defined here would not refer to the component's class, not the component
+    // instance itself.
+    _.bindAll(this, '_handleChange');
```

Comment:
There's a lot of different ways we can bind `this` to our helper functions for React callbacks. We've chosen to standardize on this conventional technique rather than using the ES7 binding or fat arrows due to different gotchas with these.

Note, you have to `_.bind(this, '_someCallback', someParam);` if you need to pass param to the callback.